optimum/intel/neural_compressor/quantization.py

```diff
@@ -25,7 +25,7 @@
 import torch
 from datasets import Dataset, load_dataset
 from neural_compressor.config import PostTrainingQuantConfig
-from neural_compressor.experimental.export import torch_to_int8_onnx
+from neural_compressor.utils.export import torch_to_int8_onnx
```

Comment:
To continue supporting previous versions of `neural-compressor` why not check its version with [`is_neural_compressor_version`](https://github.com/huggingface/optimum-intel/blob/ef23a5d44d001578e90ac2626530394178808cc8/optimum/intel/utils/import_utils.py#L309C5-L309C33)

Reply:
Yes, you are right, fixed